README-translated/README-Hindi.md

```diff
@@ -0,0 +1,127 @@
+| **समाचार** | [![Twitter Follow](https://img.shields.io/twitter/follow/turingway?style=social)](https://twitter.com/turingway) [![Join our tinyletter mailing list](https://img.shields.io/badge/receive-our%20newsletter%20❤%EF%B8%8F-blueviolet.svg)](https://tinyletter.com/TuringWay)  [![YouTube Channel Views](https://img.shields.io/youtube/channel/views/UCPDxZv5BMzAw0mPobCbMNuA?style=social)](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)  |
+| **चैट** | [![Join Slack](https://img.shields.io/badge/Chat-on%20Slack-ff69b4)](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw) [![GitHub issues](https://img.shields.io/github/issues/alan-turing-institute/the-turing-way)](https://github.com/alan-turing-institute/the-turing-way/issues) [![GitHub pull requests](https://img.shields.io/github/issues-pr/alan-turing-institute/the-turing-way)](https://github.com/alan-turing-institute/the-turing-way/pulls) |
+| **संकलन** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
+| **मुलाकात** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |
+
+*"द ट्यूरिंग वे" परियोजना के गीटहब (GitHub) भंडार में आपका स्वागत है। यहां सभी परियोजना घटकों का विकास, समीक्षा और रख-रखाव किया जाता है।.*
+
+_द ट्यूरिंग वे_ प्रतिलिपि प्रस्तुत करने योग्य, नैतिक और सहयोगी डेटा विज्ञान के लिए एक व्यावहारिक मार्गदर्शिका है। 
+हम सभी के लिए सुलभ, समझने योग्य और प्रभावी डेटा विज्ञान प्राप्त करने के लिए योगदानकर्ताओं के एक विविध समुदाय का समर्थन करते हैं। 
```

Comment:
```suggestion
हम सभी के लिए सुलभ, समझने योग्य और प्रभावी डेटा विज्ञान प्राप्त करने के लिए योगदानकर्ताओं के एक विविध समुदाय का समर्थन करते हैं। 
```
just reordering the 'hum' closer to the verb to make reading easier...
सभी के लिए सुलभ, समझने योग्य और प्रभावी डेटा विज्ञान प्राप्त करने के लिए योगदानकर्ताओं के एक विविध समुदाय का हम समर्थन करते हैं।